kernel/pipedev.c

```diff
@@ -169,6 +188,11 @@ static int _pd_pipe2(myst_pipedev_t* pipedev, myst_pipe_t* pipe[2], int flags)
         rdpipe->fd = fds[0];
         rdpipe->mode = O_RDONLY;
         rdpipe->shared = shared;
+        rdpipe->flags = flags;
```

Comment:
Should the file status flag include O_CLOEXEC?

Reply:
On line 156, is it necessary to expose O_CLOEXEC bit setting to the host?

Reply:
Fixed. The file status no longer includes O_CLOEXEC. Also, we mask off O_CLOEXEC before passing the flag to the host now.